bool stop               # flag to stop an ongoing calibration
---
bool output             # error indicator